flake.nix

```diff
@@ -8,8 +8,12 @@
     beckn-gateway.inputs.common.follows = "common";
     beckn-gateway.inputs.shared-kernel.follows = "shared-kernel";
 
-    easy-purescript-nix.url = "github:justinwoo/easy-purescript-nix";
-    easy-purescript-nix.flake = false;
+    # Frontend inputs
+    purifix.url = "github:purifix/purifix";
+    dream2nix.url = "github:nix-community/dream2nix";
+    dream2nix.inputs.nixpkgs.follows = "dream2nix-nixpkgs-fixed";
+    dream2nix-nixpkgs-fixed.url = "nixpkgs/c1989c17e2658f721df7b0027cbc3d8959f914cb"; # when this fix gets upstreamed remove
```

Comment:
please add a link to the github issue here, so as to remind us when to remove it in future.

Reply:
You can just link to https://github.com/nix-community/dream2nix/issues/522